shiptendance/app.js

```diff
@@ -0,0 +1,77 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+
+var routes = require('./routes/index');
+var users = require('./routes/users');
+
```

Comment:
Remember, according to the prompt you want all these routes to be prefixed with /pirates, so you know what entity you're working with in the URL. Also, you can delete the users since it's not being used in this app.